doc/api/http.md

```diff
@@ -1,4 +1,4 @@
-# HTTP
+﻿# HTTP
```

Comment:
There's a weird space at line 559 that made my editor decide it's a UTF-8 file, and added the BOM.